agenthub/codeact_agent/codeact_agent.py

```diff
@@ -213,7 +218,17 @@ def step(self, state: State) -> Action:
             ],
             temperature=0.0,
         )
-        return self.action_parser.parse(response)
+        action = self.action_parser.parse(response)
+        if is_autonomous:
+            user_msg = (
+                '\n----------\n'
+                'Please continue working on the task on whatever approach you think is suitable.\n'
+                'If you think you have solved the task, you can give <finish> to end the interaction.\n'
+                'IMPORTANT: YOU SHOULD NEVER ASK FOR HUMAN HELP.\n'
+            )
+            action.content += user_msg  # type: ignore
+            action.wait_for_response = False  # type: ignore
+        return action
```

Comment:
I think you probably don't need to change these on the backend, since you are already dispatching user messages on the FE?

Reply:
Not dispatched.

Reply:
Can you remove changes from the backend so that it won't affect other things (e.g., eval), and move this message to dispatched by the frontend?

Ideally, the "auto mode" should just be a way for the user to say "i want to keep clicking the [continue] button when it comes up"

Reply:
eval is not affected by this.